docs/samples/specification/azure_key_credential/readme.md

```diff
@@ -0,0 +1,16 @@
+# Sample Azure Key Credential Generation
+
+Use the flag `--credential-types` to specify you want your credential to be of type [`AzureKeyCredential`][azure_key_credential].
+
+### Settings
+
+``` yaml
+input-file: ../../../../node_modules/@microsoft.azure/autorest.testserver/swagger/head.json
```

Comment:
Maybe better just use github ULR for the json?

`../../../../node_modules/@microsoft.azure` would likely not be there until user do "npm install".

Reply:
Sounds good, haven't actually started running this code, but good point